docs/additional-functionality/rapids-udfs.md

```diff
@@ -162,11 +162,11 @@ between two float vectors using [native code](../../udf-examples/src/main/cpp/sr
 
 - [URLDecode](../../udf-examples/src/main/java/com/nvidia/spark/rapids/udf/hive/URLDecode.java)
 implements a Hive simple UDF using the
-[Java APIs of RAPIDS cudf](https://docs.rapids.ai/api/cudf-java/stable)
+[Java APIs of RAPIDS cudf](https://docs.rapids.ai/api/cudf-java/legacy)
```

Comment:
This is not the correct link.

Reply:
fixed by https://github.com/NVIDIA/spark-rapids/pull/4771